src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -694,6 +695,43 @@ public function addProductAction(int $orderId, Request $request): Response
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))", redirectRoute="admin_orders_index")
+     *
+     * @param int $orderId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function productPricesAction(int $orderId, Request $request): Response
```

Comment:
I cannot guess what this method is for 🤔  could we improve the naming ? it seems to return ... some of the informations of the product lists of an order ?

Reply:
It returns the product prices from an order indeed, it is used to refresh the product prices in the order view page

I think the naming is appropriate (OrderController::productPricesAction matches with its action/behaviour) But I'm probably too familiar with to miss the confusion Do you gave a suggestion for better naming?